### ISSUE TITLE
awesome list

### ISSUE THREAD

**Issue by edzob** [kautukkundan/Awesome-Profile-README-templates#188]
Is it an idea to add an readme including all the links to the source material?
For example: https://github.com/matiassingers/awesome-readme

**Comment by kautukkundan** [comment]
Yes, I'll planning to change the overall format of the repo to be more consistent with other "awesome" repository format. 
Just caught up with some work, will do it very soon. Thanks for the feedback 😊

**Comment by edzob** [comment]
If you can share  what you have in mind I can try to help

Edzo

-- 
"It is amazing what you can accomplish if you do not care who gets the
credit." - Harry S Truman

Mobile, Signal, WhatsApp - +31629485425
Twitter, riot.im, wire - @edzob
threma - 6FVWM3Z5
Mastodon - @edzob@hostux.social
Matrix - @edzob:matrix.org
